packages/core/src/internal-components/overlay/overlay.element.ts

```diff
@@ -56,8 +72,25 @@ type CloseChangeSources = 'backdrop-click' | 'escape-keypress' | 'close-button-c
  * @event closeChange - Notify user when close event has occurred
  * @cssprop --backdrop-background
  * @cssprop --layered-backdrop-background
+ * @cssprop --animation-duration
+ * @cssprop --animation-easing
+ *
+ * KNOWN ISSUE: Safari jumps through the exit animation but only when the ESC key is pressed.
```

Comment:
And we have a known issue where safari jumps to the end of an exit animation but only when the escape key is pressed.